_layouts/home.html

```diff
@@ -71,13 +71,19 @@ <h2>Formats</h2>
           <div class="card-media">{% include icon__format--video.svg %}</div>
         </div>
       </div>
+      <div class="card text-center">
+        <div class="card-body">
+          <a class="card-title stretched-link" href="/formats/ctv.html">Connected TV</a>
+          <div class="card-media"><img src="/assets/images/homepage/ctv_grey.png" alt="ctv"></div>
```

Comment:
Because this is a png, the hover effect seems to be different.  The image doesn't turn white, but stays grey.

I'll ask marketing if we can have the assets as SVGs as well. This is also more performant, as it can SVGs can be inlined directly into html, like the other graphics

Reply:
I noticed this, but since it took 6 months to get the new graphics, I figured it would take 6 more months to get an SVG version of the graphics. Figured the CTV committee would prefer to get the info out into the world even with a slightly disheveled appearance and fix it in the background.

Reply:
Uiiii... got the logo as an SVG and fixed the coloring myself